test/parallel/test-require-extensions-same-filename-as-dir.js

```diff
@@ -23,8 +23,10 @@
 const common = require('../common');
 const assert = require('assert');
 
-const content = require(common.fixturesDir +
-  '/json-with-directory-name-module/module-stub/one/two/three.js');
+const fixturesDir = common.fixturesDir;
```

Comment:
This is Ok.
But it would be much better to use `path.join` instead, see https://github.com/nodejs/node/pull/14272#discussion_r127608804
You could even do it with multiple parts
```js
const filePath = path.join(
  common.fixturesDir,
  'json-with-directory-name-module',
  'module-stub',
  'one-trailing-slash',
  'two',
  'three.js'
);
```